README.md

```diff
@@ -28,7 +28,7 @@ Features:
 
 ## Install
 
-1. Download latest [/releases](/releases)
+1. Download latest [/releases](https://github.com/telicent-oss/ies-syntax-highlighter-vscode-extension/releases)
```

Comment:
Suggestions button disappeared, but not sure the slash needs to be visible:
Download latest [releases](https://github.com/telicent-oss/ies-syntax-highlighter-vscode-extension/releases)